bears/general/FilenameBear.py

```diff
@@ -71,6 +77,18 @@ def run(self, filename, file,
                      file_naming_convention)
             return
 
+        if not filename_without_extension.startswith(prefix):
+            yield Result.from_values(
+                origin=self,
+                message='Filename does not use the {} prefix.'.format(prefix),
+                file=filename)
+
+        if not filename_without_extension.endswith(suffix):
+            yield Result.from_values(
+                origin=self,
+                message='Filename does not use the {} suffix.'.format(suffix),
+                file=filename)
```

Comment:
See https://github.com/coala/coala/blob/master/coalib/results/Diff.py :) There's a `rename` field you can use for this purpose ;)